pkg/clusterresource/controller.go

```diff
@@ -573,40 +585,50 @@ func (c *controller) Sync(ctx context.Context) error {
 		errs = append(errs, err)
 	}
 
+	stats := ResourceSyncStats{}
+
 	for _, project := range projects.Projects {
 		for _, domain := range project.Domains {
 			namespace := common.GetNamespaceName(c.config.NamespaceMappingConfiguration().GetNamespaceTemplate(), project.Id, domain.Name)
 			customTemplateValues, err := c.getCustomTemplateValues(
 				ctx, project.Id, domain.Id, domainTemplateValues[domain.Id])
 			if err != nil {
-				logger.Warningf(ctx, "Failed to get custom template values for %s with err: %v", namespace, err)
+				logger.Errorf(ctx, "Failed to get custom template values for %s with err: %v", namespace, err)
 				errs = append(errs, err)
 			}
-			err = c.syncNamespace(ctx, project, domain, namespace, templateValues, customTemplateValues)
+
+			newStats, err := c.syncNamespace(ctx, project, domain, namespace, templateValues, customTemplateValues)
 			if err != nil {
 				logger.Warningf(ctx, "Failed to create cluster resources for namespace [%s] with err: %v", namespace, err)
 				c.metrics.ResourceAddErrors.Inc()
 				errs = append(errs, err)
 			} else {
 				c.metrics.ResourcesAdded.Inc()
 				logger.Debugf(ctx, "Successfully created kubernetes resources for [%s]", namespace)
+				stats.Add(newStats)
 			}
 		}
 	}
+
+	logger.Infof(ctx, "Completed cluster resource creation loop with stats: %+v", stats)
```

Comment:
Do you think it might be useful to log these stats per namespace as well too, since it's easier to reason about the deltas in created/updated at the template level than remembering they get applied n times across all namespaces?